components/org.wso2.carbon.identity.user.export.core/src/main/java/org/wso2/carbon/identity/user/export/core/internal/service/impl/AbstractUserInformationProvider.java

```diff
@@ -19,9 +19,38 @@
 package org.wso2.carbon.identity.user.export.core.internal.service.impl;
 
 import org.wso2.carbon.identity.core.handler.AbstractIdentityHandler;
+import org.wso2.carbon.identity.user.export.core.UserExportException;
+import org.wso2.carbon.identity.user.export.core.internal.UserProfileExportDataHolder;
 import org.wso2.carbon.identity.user.export.core.service.UserInformationProvider;
+import org.wso2.carbon.user.api.UserRealm;
+import org.wso2.carbon.user.api.UserStoreException;
+import org.wso2.carbon.user.core.UserStoreManager;
+import org.wso2.carbon.user.core.common.AbstractUserStoreManager;
+import org.wso2.carbon.user.core.service.RealmService;
 
 public abstract class AbstractUserInformationProvider extends AbstractIdentityHandler implements
         UserInformationProvider {
 
+
+    protected static final String WSO2_CLAIM_URI = "http://wso2.org/claims/";
+    protected static final String WSO2_IDENTITY_CLAIM_URI = "http://wso2.org/claims/identity/";
+    protected static final String WSO2_RUN_TIME_CLAIM_URI = "http://wso2.org/claims/runtime/";
+
+    protected UserStoreManager getUserStoreManager(int tenantId, String userStoreDomain) throws UserExportException {
+
+        AbstractUserStoreManager userStoreManager;
+        try {
+            RealmService realmService = UserProfileExportDataHolder.getRealmService();
+            UserRealm userRealm = realmService.getTenantUserRealm(tenantId);
```

Comment:
Is it possible to userRealm to be null?

Reply:
According to the implementation of these methods, there seems to be no possibility. And this seems to be the common practice in handling these 2 objects in the code base.